_includes/configuration/features.html

```diff
@@ -72,6 +72,11 @@
       <td>Whether the web based intake is supported or not</td>
     </tr>
     <tr>
+      <td class="inlinecode">intakeIpmiOptional</td>
+      <td>Boolean</td>
+      <td>Whether the tumblr intake process may skip UID verification for assets without IPMI configuration.</td>
```

Comment:
I only found a reference in collins/conf/reference/features_reference.conf, but not a big deal either way, even though it should be reworked too.